crates/oci/Cargo.toml

```diff
@@ -25,7 +25,7 @@ spin-manifest = { path = "../manifest" }
 spin-trigger = { path = "../trigger" }
 tempfile = "3.3"
 tokio = { version = "1", features = ["fs"] }
-tokio-util = "0.7.9"
+tokio-util = { version = "0.7.9", features = ["compat"] }
```

Comment:
This cropped up seemingly with the oci-distribution version bump, oddly.  (`compat` not found unless declared as a feature).  I can see about PRing it separately...

Reply:
(Added in https://github.com/fermyon/spin/pull/1883)